crates/p2p/src/swarm/mod.rs

```diff
@@ -341,7 +342,7 @@ impl P2P {
             .source
             .expect("Message must have author as ValidationMode set to Permissive");
         if let Err(err) = self.validate_gossipsub_msg(&msg) {
-            error!(reason=%err, %source, "Message failed validation.");
+            warn!(reason=%err, %source, "Message failed validation.");
```

Comment:
can we get some consistency on capitalization? I think we should go for all lowercase and no trailing periods. idc about exactly what the rules are but consistency would be good.

Reply:
Will do a follow-up PR changing all log messages to lower case.